docker-compose.yml

```
@@ -43,8 +43,8 @@ services:
             - cassandra
     mysql:
         container_name: streamr-dev-mysql
-        platform: linux/x86_64
-        image: mysql:5.7
+        # platform: linux/x86_64
+        image: mysql #:5.7
```

Could use explicit version. This seems to work:

```suggestion
        image: mysql:9.1.0
```